ui/perfherder/alerts/StatusDropdown.jsx

```diff
@@ -216,6 +254,57 @@ export default class StatusDropdown extends React.Component {
     alertStatus === 'investigating' ||
     (alertStatus !== status && this.isResolved(alertStatus));
 
+  calculateDueDate(created) {
+    const createdAt = new Date(created);
+    const dueDate = new Date(created);
+
+    /* due date is calculated by adding 3 working days to the date when the alert was created
+    this means you have 3 working days with decreasing status of 3, 2, 1 working days left
+    and in the fourth day the status changes to Today, this being the last day when the alert status has to change
+    otherwise it becomes Overdue */
+
+    dueDate.setDate(dueDate.getDate() + timeToTriage);
+    const numberOfNonWorkingDays = this.getNumberOfWeekendDays(
+      createdAt,
+      dueDate,
+    );
+
+    if (numberOfNonWorkingDays !== 0) {
+      dueDate.setDate(dueDate.getDate() + numberOfNonWorkingDays);
+    }
+
+    return dueDate;
+  }
+
+  displayDueDateCountdown(createdAt) {
+    const now = new Date(Date.now());
+    const dueDate = this.calculateDueDate(createdAt);
+
+    const differenceInTime = Math.abs(dueDate - now);
+    const differenceInDays = Math.ceil(
+      differenceInTime / (1000 * 60 * 60 * 24),
+    );
+
+    // if the website is accessed during the weekend, nothing will be shown
+    if (now.getDay() === 6 || now.getDay() === 0) {
```

Comment:
This solution is not ideal, in my opinion the tooltip should not be shown at all during the weekend. This is what it will be displayed on Saturday or Sunday:
![timetotriage](https://user-images.githubusercontent.com/69891317/153177067-e226d4cd-8702-43e2-8ea7-eab5bbaa4ef1.png)
l

Reply:
Also, if you return '' the css `due-date-overdue` and `due-date-today` will never be used during the weekend and the color shown for `faClock` will always be green.